includes/blocks/class-xpay-blocks-support.php

```
@@ -0,0 +1,64 @@
+ * XPay_Gateway), so the Blocks surface stays minimal: render the
+ * title/description, and let Blocks' standard redirect flow carry the
+ * shopper to the pay page. One flow for both checkouts, by design.
+ *
+ * @package XPay_For_WooCommerce
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+use Automattic\WooCommerce\Blocks\Payments\Integrations\AbstractPaymentMethodType;
+
+final class XPay_Blocks_Support extends AbstractPaymentMethodType {
```

PHPStan (2.2.7)

[error] Class XPay_Blocks_Support extends unknown class Automattic\WooCommerce\Blocks\Payments\Integrations\AbstractPaymentMethodType.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)